style.css

```diff
@@ -351,3 +364,181 @@ textarea {
   color: #fff;
   background-color: #3347ff;
 }
+
+@media (min-width: 600px) {
+  .mobile {
+    display: none;
+  }
+
+  .desktop {
+    display: flex;
+  }
+
+  .nav-bar .fa-bars {
+    display: none;
+  }
+
+  .header {
+    height: 100vh;
+    background-image: url(./images/desktop\ version\ background.png);
+    background-position-y: 0;
+    border-bottom-left-radius: 70px;
+    margin-top: 0;
+  }
+
+  .nav-bar {
+    position: absolute;
+    left: 0;
+    top: 0;
+    padding: 1rem 10% 1rem 10%;
+    background-color: #fff;
+  }
+
+  .header .nav-list {
+    display: flex;
+    align-items: center;
+  }
+
+  .nav-list li {
+    margin-right: 1.5rem;
+  }
+
+  .nav-list a {
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .container {
+    width: 60%;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  .container .br {
+    display: none;
+  }
+
+  .icons {
+    display: flex;
+    justify-content: space-between;
+    width: 20%;
+  }
+
+  /* portfolio  */
+  .project {
+    box-sizing: border-box;
+    width: 100%;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding: 14px 0;
+  }
+
+  .project:nth-child(even) {
+    flex-direction: row-reverse;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .project-img {
+    width: 45%;
+    margin-left: 24px;
+  }
+
+  .project-img:nth-child(even) {
+    margin-left: 100px;
+  }
+
+  .projects-container {
+    margin: 100px 150px;
+    display: grid;
+    grid-template-rows: repeat(4, 2fr);
+    gap: 55px;
+    padding: 24px;
+  }
+
+  .project-txt {
+    width: 45%;
+    padding-top: 1rem;
+  }
+
+  /* About content */
+
+  .about {
+    padding: 114px 13%;
+    border-top-right-radius: 50px;
+  }
+
+  .about-content {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .about-text {
+    width: 50%;
+    margin-right: 7rem;
+  }
+
+  .knowledge {
+    width: 40%;
+  }
+
```

Comment:
- [ ] Kindly make sure you apply the correct width value on the `About-text` block and `knowledge` block. If you correctly check on the Figma template you will realize that the `About-text`  block has a width less than the one of the `knowledge`(e.g.:  `.about-text { width: 40%; }` and `.knowledge { width: 60%; }`). Make sure you fix this issue to stick to the Figma design.